server/src/main/java/org/candlepin/resource/OwnerContentResource.java

```diff
@@ -289,6 +298,12 @@ public ContentData updateContent(@PathParam("owner_key") String ownerKey,
         }
 
         existing = this.contentManager.updateContent(content, owner, true);
+        try {
+            ownerManager.refreshOwnerForContentAccess(owner);
```

Comment:
Doing this during every content update could lead to pretty ugly performance implications if there are a lot of consumers. What is the performance of this if an owner has 40k consumers?